File: Ambulance.BLL/Models/BrigadeResourceModel.cs
﻿namespace Ambulance.BLL.Models;

public class BrigadeResourceModel
{
    public int BrigadeId { get; set; }
    public string? BrigadeType { get; set; }
    public int TotalCallsHandled { get; set; }
    public int DistinctItemsUsed { get; set; }
}

File: ambulance-site/src/api/models/AnaliticsModels/BrigadeResourceModel.js
// models/brigadeResourceModel.js
export class BrigadeResourceModel {
    constructor({ brigadeId = 0, brigadeType = null, totalCallsHandled = 0, distinctItemsUsed = 0, averageCallDurationMinutes = 0.0 } = {}) {
        this.brigadeId = brigadeId;
        this.brigadeType = brigadeType;
        this.totalCallsHandled = totalCallsHandled;
        this.distinctItemsUsed = distinctItemsUsed;
        this.averageCallDurationMinutes = averageCallDurationMinutes;
    }
}
